solutions/standard/variables.tf

```diff
@@ -237,3 +250,52 @@ variable "skip_iam_authorization_policy" {
   description = "Whether to create an IAM authorization policy that permits all PostgreSQL instances in the resource group to read the encryption key from the Hyper Protect Crypto Services instance specified in the `existing_kms_instance_crn` variable."
   default     = false
 }
+
+##############################################################
+# Backup Encryption
+##############################################################
+variable "backup_key_name" {
+  type        = string
+  default     = "postgresql-backup-key"
+  description = "The name for the key created for the PostgreSQL key. Applies only if not specifying an existing key. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+}
+
+variable "backup_key_ring_name" {
+  type        = string
+  default     = "postgresql-backup-key-ring"
+  description = "The name for the key ring created for the PostgreSQL key. Applies only if not specifying an existing key. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+}
```

Comment:
I would not expose `backup_key_name` or `backup_key_ring_name`. Just use the values of `key_name` and `key_ring_name` for backup keys too